jsonifier/models/models.py

```diff
@@ -194,6 +201,62 @@ def jsonify(self, parser, one=False):
              'field_name:json_key'
         ]
 
+        if with_fieldname=True,the name of the field will be added
+        to the value. the name of the field will be translated.
+
+        rec.jsonify(
+            parser=['name', 'create_date',
+                    'order_line', ['id' , 'product_uom', 'is_expense'])],
+            with_fieldname=False)
+
+        returns:
+
+        [{'name': 'SO/2022/006',
+                   'create_date': '2022-10-14T15:55:39.823019+02:00',
+                   'order_line': {
+                         'id': 4,
+                         'product_uom': 'Stuk(s)
+                         'is_expense': False}
+                    }]
```

Comment:
I don't feel that this is the place for an elaborate documentation of the usage of this module with examples, that's for `readme/USAGE.txt`. It also needs an extra line that explains the use cases in which this is necessary, and that's when you are using `jsonifier` in the context of an API, and you want to build a visual UI around the nested dataset that you are fetching - then you need the field names.